tools/warnings_count/warnings_setup_current.txt

```diff
@@ -1,7 +1,4 @@
-In file included from /usr/include/string.h:519,
-                 from vtxdis.c:5:
-In function ‘strncpy’,
-    inlined from ‘parse_int’ at vtxdis.c:41:9:
-/usr/include/x86_64-linux-gnu/bits/string_fortified.h:106:10: warning: ‘__builtin_strncpy’ specified bound 20 equals destination size [-Wstringop-truncation]
-  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
-      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+src/rom.c: In function ‘rom_dma_repack’:
+src/rom.c:1516:4: warning: format not a string literal and no format arguments [-Wformat-security]
+ 1516 |    die(errstr);
+      |    ^~~
```

Comment:
You fixed this warning in z64compress, right? If that's the case, it could be removed without problems.
To update the warnings you could run `./tools/warnings_count/update_current_warnings.sh` (may be slow tho)